.github/workflows/test.yml

```diff
@@ -1,11 +1,15 @@
+name: Test
+
 on:
   push:
     branches:
       - main
   pull_request:
-    branches:
-      - main
```

Comment:
Interesting this was only for main branch.

Reply:
Weird.